vita3k/gui/src/content_manager.cpp

```diff
@@ -590,9 +590,7 @@ void draw_content_manager(GuiState &gui, HostState &host) {
                     content.second = false;
             }
         }
-        const auto is_enable = std::find_if(contents_selected.begin(), contents_selected.end(), [&](const auto &cs) {
-            return cs.second;
-        }) != contents_selected.end();
+        const auto is_enable = boost::algorithm::any_of(contents_selected, [&](const auto &cs) { return cs.second; });
```

Comment:
Increasing dependency on Boost when `std::algorithm::any_of` already exists is redundant. Are there any advantages over using the `std` implementation?

Reply:
There is no difference compared to the `std` version except its function signature / parameters.